docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,150 @@
+---
+
+<Alert level="info">
+  The Dev Toolbar is currently in **beta**. Beta features are still in progress and may have bugs. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
+</Alert>
+
+## Set Up Your Web App
+
+<Alert level="warning">
+    [Enabling tracing](/platforms/javascript/tracing/) is a prerequisite using the Developer Toolbar. Tracing is used to collect page-specific issues and feedback through transactions.
+</Alert>
+
+You need to complete two steps to get the toolbar rendered on the page:
+1. Add or dynamically inject the following script into your web app:
+    ```html
+    <script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+    ```
+    It is recommended that you add the script tag at the bottom of the page so it doesn’t block other critical JavaScript.
+
+2. Call `window.SentryToolbar.init(initConfig)` to set up a toolbar instance on each page where you want to see the Dev Toolbar.
+    ```html
+    <html>
+    <head>...</head>
+    <body>
+        …
+        <script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+        <script>
+        window.SentryToolbar.init({ … });
+        </script>
+    </body>
+    </html>
+    ```
+
+### Unmounting The Toolbar
+
+If you have called `SentryToolbar.init({...})` to render the toolbar, but now want to manually remove or unmount it from the page, you can call the cleanup function that is returned from `init()`. This will unmount all the injected HTML and CSS. Login credentials will not be removed, so you can re-insert the toolbar and still be authenticated.
+```html
+<script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+<script>
+  const unmountToolbar = window.SentryToolbar.init({ … });
+
+  // sometime later...
+  unmountToolbar();
+</script>
+```
+
+## Init Configuration Fields
+
+The following fields can be passed into the `.init()` function.
+
+At minimum, you should be calling `.init()` with these three fields:
+```javascript
+window.SentryToolbar.init({
+  organizationSlug: 'acme',
+  projectIdOrSlug: 'website',
+  environment: 'production'
+});
+```
+And you can include any additional fields from this list:
+
+| Field Name | Type | Description | Default Value |
+| ----- | ----- | ----- | ----- |
+| `organizationSlug` | `string` | The organization that users should login to. For example \'acme\' | *Required Value* |
+| `projectIdOrSlug` | `string \| number` | The project for which this website/webapp is associated. | *Required Value* |
+| `environment (optional)` | `string \| string[] \| undefined` | The environment of this deployment. Used to narrow search results in the Toolbar UI. Set to `undefined` or `""` or `[]` if you want to see results from all environments. | `undefined` |
```

Comment:
```suggestion
| `environment (optional)` | `string \| string[] \| undefined` | The environment of this deployment. Used to narrow search results in the Toolbar UI. Set to `undefined` or `""` or `[]` if you want to see results from all environments. | `undefined` |
```

Do we need to share all the different ways to set this to see results for all environments ('undefined', '""', or '[]'). It would simplify things if we just picked one.